### PR TITLE
Fixed display element on small device

### PR DESCRIPTION
**Pull request by magsout**
- Position Header element on small device
header

- Position features on small device
features

- LIne height list of tools on small device

usage

### CODE REVIEW

**Comment by MoOx** [comment]
Can you provide screenshots before/after please ?

File: docs/content/index.md [hunk 1]
```diff
@@ -46,67 +46,67 @@ title: cssnext
   <div class="r-Grid-cell r-minM--2of3">
   <h2 class="cssnext-Section-title"><a href="/features/">Features</a></h2>
     <ul class="r-Grid cssnext-List cssnext-FeaturesList">
-      <li class="r-Grid-cell r-all--1of2">
+      <li class="r-Grid-cell r-all--1of2 r-maxS--2of2">
```
Comment (MoOx) [inline docs/content/index.md]: Hum, `2of2` seems to be 100%, which is the default value right ? Maybe we should replace `r-all--1of2` by `r-minL--1of2` instead (or minM) ?

File: docs/src/index.css [hunk 2]
```diff
@@ -294,6 +298,7 @@ h3 {
 .cssnext-Tools {
   text-align: center;
   font-size: var(--r-h3-fontSize);
+  line-height: 1.44;
```
Comment (MoOx) [inline docs/src/index.css]: wat iz dat value ? :D

Comment (Macxim) [inline docs/src/index.css]: Magic number!
giphy

Comment (magsout) [inline docs/src/index.css]: no, var(--r-h3-fontSize) without rem ;p

Comment (MoOx) [inline docs/src/index.css]: What about `var(--r-h3-lineHeight)` ?

File: docs/content/index.md (hunk 1 above)
Comment (magsout) [inline docs/content/index.md]: .r-minS--1of2,is better, no ?

Comment (MoOx) [inline docs/content/index.md]: probably :D

**Comment by magsout** [comment]
Done

**Comment by MoOx** [comment]
w00t.

**Comment by MoOx** [comment]
Patchs online in a minute :)

**Comment by Macxim** [comment]
Way better :+1:

**Pull request merged**